.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
-Cargo.lock
\ No newline at end of file
+Cargo.lock
+/.vscode
```

Comment:
```suggestion
```
Please don't add editor-specific entries to the repository's gitignore file.
You should use something like [`core.excludesFile`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreexcludesFile) to do this instead.